src/hooks/focusInput.ts

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const focusInput = () => {
```

Comment:
nit: my understanding is that there's a convention that hooks use a name that look like `useXXX`. What about `useFocusInput` for this one?

Also it would be good to have a comment explaining what the hook does, with an example of how to use it.